go-controller/pkg/ovn/base_network_controller_egressip.go

```diff
@@ -2309,3 +2381,14 @@ func getPodNamespaceAndNameFromKey(podKey string) (string, string) {
 	parts := strings.Split(podKey, "_")
 	return parts[0], parts[1]
 }
+
+// getFirstNonNilNamespace must return a non-nil pointer to a Namespace otherwise panic
+func getFirstNonNilNamespace(n1, n2 *v1.Namespace) *v1.Namespace {
+	if n1 != nil {
+		return n1
+	}
+	if n2 != nil {
+		return n2
+	}
+	panic("getNonNilNamespace(): received two nil Namespace pointers")
```

Comment:
we should never get into this scenario where theres two nil namespaces passed and if we do I wanted to scream via a panic instead of it possibly hidden in error logs. WDYT?

Reply:
Agree, crashing the controller is harsh. Removed - not needed anymore